packages/vscode-extension/src/builders/buildAndroid.ts

```diff
@@ -65,12 +66,15 @@ export async function getAndroidBuildPaths(
   return { apkPath, packageName };
 }
 
-function makeBuildTaskName(productFlavor: string, buildType: string) {
-  // task name is in the format of assemble<ProductFlavor><BuildType> where productFlavor and buildType
+function makeBuildTaskName(productFlavor: string, buildType: string, appName?: string) {
+  // task name is in the format of :<appName>:assemble<ProductFlavor><BuildType> where productFlavor and buildType
   // are the names of the productFlavor and buildType that each start with a capital letter.
   // By default, Android Gradle Plugin always creates staging and release buildTypes and does not define any productFlavor.
+  // and appName is the name of the application as seen by the gradle build system, omitting appName leads to
+  // whole android file being build.
```

Comment:
Pedantic:
```suggestion
  // whole android project being build.
```
?
I don't know what the correct nomenclature here is, but "file" doesn't feel right.